src/Extensions/Abstractions/Activities/Processing/ActivityTagKeys.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions.Activities.Processing
+{
+	/// <summary>
+	/// Activity does not have methods to extract tags efficiently, so we need to expose keys to provide ability of extracting tags
+	/// </summary>
+	public static class ActivityTagKeys
+	{
+		/// <summary>
+		/// Activity result tag key
+		/// </summary>
+		public static string Result { get; } = "Result";
+
+		/// <summary>
```

Comment:
Nit: add another blank line for consistent formatting

Reply:
Added, thanks, I'll create pr for changing formating after merging this one